test/Tests.Infrastructure/RavenRetryFactAttribute.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using xRetry;
+using Xunit.Sdk;
+
+namespace Tests.Infrastructure;
+
+[TraitDiscoverer("Tests.Infrastructure.XunitExtensions.RavenTraitDiscoverer", "Tests.Infrastructure")]
+public class RavenRetryFact : RetryFactAttribute, ITraitAttribute
```

Comment:
Instead of introducing a new attribute we can do the following:
https://github.com/ravendb/ravendb/pull/18370/files#r1560647376

Reply:
Please note we'll need that for Fact and Theory as well

Reply:
decided to add additional attribute for `RavenRetryFact` / `RavenRetryTheory`